tc/core/polyhedral/schedule_tree.h

```diff
@@ -124,6 +124,12 @@ struct ScheduleTree {
       : ctx_(ctx), type_(type), elem_(std::move(elem)) {
     appendChildren(std::move(children));
   }
+
+  // Copy constructor for internal use only.
+  // Note that this does not copy the underlying elem_.
+  // All callers should use makeScheduleTree(const ScheduleTree&) instead,
+  // which copies the underlying elem_ as well as maintains the memory
+  // management scheme consistent.
```

Comment:
This sounds weird (possibly French?).
Possible alternatives:
"keeps the memory management scheme consistent"
"maintains the memory management scheme"
"maintains a consistent memory management scheme"